common/common-db/src/main/java/org/ow2/proactive/db/TransactionHelper.java

```diff
@@ -124,6 +126,8 @@ public TransactionHelper(SessionFactory sessionFactory) {
             session.getTransaction().commit();
 
             return result;
+        } catch (DatabaseManagerException | IllegalArgumentException e) {
+            throw e;
```

Comment:
@fviale I believe here I have to rollback as well. DME and IAE are both thrown by us, but transaction still has to be rollbacked. Isn't it?